pkgs/development/libraries/bamf/default.nix

```diff
@@ -1,54 +1,66 @@
-{ stdenv, fetchurl, libgtop, libwnck3, glib, vala, pkgconfig
-, libstartup_notification, gobjectIntrospection, gtk-doc
+{ stdenv, autoconf, automake, libtool, gnome3, which, fetchgit, libgtop, libwnck3, glib, vala, pkgconfig
+, libstartup_notification, gobjectIntrospection, gtk-doc, docbook_xsl
 , xorgserver, dbus, python2 }:
 
 stdenv.mkDerivation rec {
-  pname = "bamf";
-  version = "0.5.3";
-  name = "${pname}-${version}";
+  name = "bamf-2018-02-07";
 
   outputs = [ "out" "dev" "devdoc" ];
 
-  src = fetchurl {
-    url = "https://launchpad.net/${pname}/0.5/${version}/+download/${name}.tar.gz";
-    sha256 = "051vib8ndp09ph5bfwkgmzda94varzjafwxf6lqx7z1s8rd7n39l";
+  src = fetchgit {
+    url = https://git.launchpad.net/~unity-team/bamf;
+    rev = "0.5.3+18.04.20180207.2-0ubuntu1";
+    sha256 = "0hvbgzi0mzzzvcamd9mi1ykbk2l6zxffspyk5fpik8bij56nhzym";
   };
 
   nativeBuildInputs = [
-    pkgconfig
-    gtk-doc
+    autoconf
+    automake
+    docbook_xsl
+    gnome3.gnome-common
     gobjectIntrospection
+    gtk-doc
+    libtool
+    pkgconfig
     vala
+    which
     # Tests
-    xorgserver
+    python2
```

Comment:
Does python3 not work?

Reply:
```
error: libxslt not supported for interpreter python3.6m
```

Reply:
Hmm, we only fixed libxml2 :-/ https://github.com/NixOS/nixpkgs/pull/48609